dev/defs/constraints/transform_space_constraint.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "TransformSpaceConstraint",
+  "key": {
+    "int": 90,
+    "string": "transformspaceconstraint"
+  },
+  "abstract": true,
+  "extends": "constraints/targeted_constraint.json",
+  "properties": {
+    "sourceSpaceValue": {
+      "type": "uint",
+      "initialValue": "0",
+      "key": {
+        "int": 179,
```

Comment:
Yep, another property gets them now, it all works out because coop still has them stored against the concrete object, as long as it inherits this class it'll still all work out.